pytest_mpl/plugin.py

```diff
@@ -92,48 +124,80 @@ def pytest_report_header(config, startdir):
+           'location where py.test is run.')
+    group.addoption('--mpl-generate-path', help=msg, action='store')
+
+    msg = ('Filepath to save a generated hash library, relative to '
+           'location where py.test is run.')
+    group.addoption('--mpl-generate-hash-library', help=msg, action='store')
+
+    msg = ('Directory containing baseline images, relative to location '
+           'where py.test is run unless --mpl-baseline-relative is given. '
+           'This can also be a URL or a set of comma-separated URLs '
+           '(in case mirrors are specified).')
+    group.addoption('--mpl-baseline-path', help=msg, action='store')
+
+    msg = 'Interpret the baseline directory as relative to the test location.'
+    group.addoption("--mpl-baseline-relative", help=msg, action="store_true")
+
+    msg = ('JSON library of image hashes, relative to location where '
+           'py.test is run.')
+    group.addoption('--mpl-hash-library', help=msg, action='store')
+
+    msg = ('Generate a summary report of any failed tests, in '
+           '--mpl-results-path. The type of the report should be specified. '
+           'Supported types are `html`, `json` and `basic-html`. '
+           'Multiple types can be specified separated by commas.')
+    group.addoption('--mpl-generate-summary', help=msg, action='store')
+
+    msg = ('Directory for test results, relative to location where py.test '
+           'is run.')
+    option = 'mpl-results-path'
+    group.addoption(f'--{option}', help=msg, action='store')
+    parser.addini(option, help=msg)
+
+    msg = ('Always compare to baseline images and save result images, even '
+           'for passing tests. This option is automatically applied when '
+           'generating a HTML summary.')
+    option = 'mpl-results-always'
+    group.addoption(f'--{option}', help=msg, action='store_true')
+    parser.addini(option, help=msg)
+
+    msg = 'Use fully qualified test name as the filename.'
+    parser.addini('mpl-use-full-test-name', help=msg, type='bool')
+
+    msg = ('Algorithm to be used for hashing images. Supported kernels are '
+           '`sha256` (default) and `phash`.')
+    option = 'mpl-kernel'
+    group.addoption(f'--{option}', help=msg, action='store')
+    parser.addini(option, help=msg)
+
+    msg = ('Determine the level of image detail (high) or structure (low)'
+           'represented in the perceptual hash.')
+    option = 'mpl-high-freq-factor'
+    group.addoption(f'--{option}', help=msg, action='store')
+    parser.addini(option, help=msg)
+
+    msg = 'The hash size (N) used to generate a N**2 bit image hash.'
+    option = 'mpl-hash-size'
+    group.addoption(f'--{option}', help=msg, action='store')
+    parser.addini(option, help=msg)
+
+    msg = 'Hamming distance bit tolerance for similar image hashes.'
+    option = 'mpl-hamming-tolerance'
+    group.addoption(f'--{option}', help=msg, action='store')
+    parser.addini(option, help=msg)
```

Comment:
Just tidied this section up a wee bit since I was in the space... hope that's not too controversial?